src/devices/ambient_weather_tx8300.c

```diff
@@ -0,0 +1,136 @@
+/* Ambient Weather TX-8300 Thermometer
+ * contributed by Roger
+*/
+
+#include <byteswap.h>
+#include "rtl_433.h"
+#include "data.h"
+#include "util.h"
+
+/* 
+Packet format (74 bits):
+   [2 bit preamble] [1 bit start bit] [31 bit payload] [1 bit start bit] [31 bit payload (inverted)] [8 bit CRC]   
+   HH1PPPPP PPPPPPPP PPPPPPPP PPPPPPPP PP1QQQQQ QQQQQQQQ QQQQQQQQ QQQQQQQQ QQCCCCCC CC   
+   
+Preamble format (2 bits):
+    [1 bit (0)] [1 bit rolling count]    
+    0R
+
+Payload format (31 bits):
+   [9 bit unknown (humidity?)] [2 bit channel number] [1 bit negative flag] [7 bit ID]  [12 bit BCD temperature (C, 1 decimal point)]   
```

Comment:
Best to take the payload as 32 bit and mask out the fixed top bit, humidity needs only 7 bits, might be a full byte, but havn't ever seen 9 bits

Reply:
I’m not sure anymore what that “humidity” field is anymore. Could be a combination of battery status, unused humidity or device code or something.